harness/determined/exec/prep_container.py

```diff
@@ -27,32 +27,7 @@ def is_trial(info: det.ClusterInfo) -> bool:
 
 def download_context_directory(sess: api.Session, info: det.ClusterInfo) -> None:
     context_directory_resp = None
-    try:
```

Comment:
What if someone doesn't use ship-logs?   Maybe that can't happen?

Reply:
I thought about that.  It shouldn't happen unless somebody is running in detached mode, and if they're in detached mode, they sure shouldn't expect `prep_container.py` to work for them.